components/sync15/ios/RustSyncTelemetryPing.swift

```diff
@@ -0,0 +1,622 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+public class RustSyncTelemetryPing {
+    public var version: Int
```

Comment:
nit should those be `let` and not `var`?

Reply:
See my comment below 😬